### DISCUSSION TITLE
Add a new operation that changes the toaster message shown after the flow finishes

### DISCUSSION THREAD

**Discussion by NigmaX** [directus/directus#15026] [Feature Requests]
currently, after manually triggering a flow, a toaster message will appear with the message "Flow "{flow}" ran successfully"

but it could be nice to have this message controlled from the flow's "success"/"reject" funnels.
some of the flows have API requests that might fail the entire flow. 
and still, you will get a "Flow "{flow}" ran successfully" message which can be confusing.

what is your thought about it? is it something that you guys plan to add?
Thanks!

**Comment by rijkvanzanten** [comment]
Great idea! Similarly, I'd love to allow the flow to push multiple messages during the execution, so you get a proper progress bar / log. @br41nslug one for the websockets list 😄

**Comment by azrikahar** [comment]
Similar idea here! https://github.com/directus/directus/discussions/12098#discussioncomment-2833344

**Comment by colemahatmccreary** [comment]
Here's my old idea for a fairly extensible approach to this:
https://github.com/directus/directus/discussions/14201

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
